metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -2291,7 +2291,8 @@ class MetalsLanguageServer(
   }
 
   private def slowConnectToBuildServer(
-      forceImport: Boolean
+      forceImport: Boolean,
+      isBuildChangeNotification: Boolean = false,
```

Comment:
could you add scalafoc for `isBuildChangeNotification`? In a few months it won't be clear why this parameter was added